app/api/chat-messaging/route.ts

```
@@ -228,6 +279,16 @@ export async function POST(req: NextRequest) {
       systemPrompt += `\n--- END USER PROFILE ---`;
     }
 
+    // Add global memory if enabled
+    if (globalMemory && globalMemory.enabled && globalMemory.memories.length > 0) {
+      systemPrompt += `\n\n--- MEMORY ---`;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)